src/components/Guilds/ErrorBoundary/GlobalErrorBoundary.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+class GlobalErrorBoundary extends React.Component<{}, { hasError: boolean }> {

```

Comment:
Why are we using class style react component here?

Reply:
Weirdly, error boundaries cannot be implemented with functional components as of yet.